Makefile

```diff
@@ -135,6 +136,48 @@ build-linux-with-shared-library:
 	docker cp temp:/lib/libwasmvm.so $(BUILDDIR)/
 	docker rm temp
 
+build-release: build-release-amd64 build-release-arm64
+
+build-release-amd64: go.sum $(BUILDDIR)/
+	$(DOCKER) buildx create --name core-builder || true
+	$(DOCKER) buildx use core-builder
+	$(DOCKER) buildx build \
+		--build-arg GO_VERSION=$(GO_VERSION) \
+		--build-arg GIT_VERSION=$(VERSION) \
+		--build-arg GIT_COMMIT=$(COMMIT) \
+    --build-arg BUILDPLATFORM=linux/amd64 \
+    --build-arg GOOS=linux \
+    --build-arg GOARCH=amd64 \
+		-t core:local-amd64 \
+		--load \
+		-f Dockerfile .
+	$(DOCKER) rm -f core-builder || true
+	$(DOCKER) create -ti --name core-builder core:local-amd64
```

Comment:
need to add `mkdir -p $(BUILDDIR)/release/` as well. It outputs release not found upon new creation. Same for arm64

Reply:
I saw that you added one to github workflow, but maybe added one to Makefile as well so that running it directly from local won't output error

Reply:
I guess that should be all for reviewing, otherwise I am good